tasks/build/package_scripts/pre_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if command -v systemctl >/dev/null && systemctl is-active kibana.service >/dev/null; then
+    systemctl --no-reload stop kibana.service
+fi
+if [ -x /etc/init.d/kibana ]; then
```

Comment:
I see you added the `elif` instead of the `if`. What do you think of my other comment:

> Should you also check here if Kibana is running first, before attempting to stop it?

Specifically, on line 3 you use `systemctl` to check if Kibana is running before attempting to stop it on line 4. Is there a reason not to do a similar check on line 6 with `/etc/init.d/kibana` before attempting to stop it on the subsequent lines?

Also, is there a reason the `invoke-rc.d` and `service` checks are inside the `if [ -x /etc/init.d/kibana ]` check? Why not make them sibling checks instead like so:

```
elif [ -x /etc/init.d/kibana >/dev/null && /etc/init.d/kibana status >/dev/null ]; then
    /etc/init.d/kibana stop
elif [ command -v invoke-rc.d >/dev/null && invoke-rc.d kibana status >/dev/null ]; then
    invoke-rc.d kibana stop
 elif [ command -v service >/dev/null && service kibana status >/dev/null]; then
    service kibana stop
fi
```